data_structures/skip_list.cpp

```diff
@@ -0,0 +1,197 @@
+/**
+ * A skip list is a data structure that is used for storing a sorted list of items with a
+ *  help of hierarchy of linked lists that connect increasingly sparse subsequences of the items
+ * 
+ *  References used: GeeksForGeeks skip list code, https://iq.opengenus.org/skip-list/ Pseudo Code.
+*/
+
+#include <iostream>
+#include <vector>
+#include <stdio.h>
```

Comment:
please remove C header files. This is a C++ code!

Reply:
done

Reply:
If this is still used, use `cstdio`, which is a C++ library.

Reply:
No, I've deleted it. Thanks for your comment! by the way are we going to merge this or is there anything that must be changed?